runtime/interpreter/interpreter.go

```diff
@@ -2571,6 +2571,39 @@ var converterDeclarations = []valueConverterDeclaration{
 	},
 }
 
+func lookupInterface(interpreter *Interpreter, typeID string) (*sema.InterfaceType, error) {
+	location, qualifiedIdentifier, err := common.DecodeTypeID(typeID)
+	// if the typeID is invalid, return nil
+	if err != nil {
+		return nil, err
+	}
+	if location == nil {
+		return nil, &InterfaceMissingLocation{QualifiedIdentifier: qualifiedIdentifier}
```

Comment:
```suggestion
		return nil, &InterfaceMissingLocationError{
			QualifiedIdentifier: qualifiedIdentifier,
		}
```

Also, maybe move this check to `getInterfaceType`, so it will get reported for both functions